### PR TITLE
Prometheus label / body only support some char.

### PR DESCRIPTION
**Pull request by shinji62**
Prometheus do not allow some character, the first implementation was not enough restrictive and didn't apply to label.

So I made it more restrictive and apply to label as well.

### CODE REVIEW

File: src/riemann/prometheus.clj [hunk 1]
```diff
@@ -9,10 +9,11 @@
   "A set of event fields in Riemann with special handling logic."
   #{:service :metric :tags :time :ttl})
 
+
```
Comment (jamtur01) [inline src/riemann/prometheus.clj]: Extra line?

File: src/riemann/prometheus.clj [hunk 2]
```diff
@@ -9,10 +9,11 @@
...
 (defn replace-disallowed
   "Replaces all existence of disallowed characters with underscore."
   [field]
-  (str/escape field {\space "_", \. "_", \: "_" \- "_"}))
+   (str/replace (str/replace field #"[^a-zA-Z0-9_]" "_") #"[_]{2,}" "_"))
```
Comment (jamtur01) [inline src/riemann/prometheus.clj]: Extra space?

Comment (shinji62) [inline src/riemann/prometheus.clj]: Sorry ..

File: src/riemann/prometheus.clj (hunk 1 above)
Comment (shinji62) [inline src/riemann/prometheus.clj]: sorry again ..

**Comment by shinji62** [comment]
Should be fine.
Thanks.

**Comment by jamtur01** [comment]
Awesome - I am going to take @pradeepchhetri's 👍  as subject matter approval as I don't use Prometheus.

**Comment by shinji62** [comment]
@jamtur01 any chance to release ?
Thanks

**Pull request merged**